src/sms/tokens.ts

```diff
@@ -107,7 +90,7 @@ export const smsTokenMap = {
     },
     line_breaks: true,
   }),
-  GraphQlBlock: createToken({
+  GraphQlBlock: createKeyword({
```

Comment:
This one as well is not a keyword

Reply:
gotcha, this is a block. I guess we don't care at the moment if we have an augmented block?

`GraphQlBlockoops{}`

That said, not sure what the definition of keyword is.

Reply:
A keyword is a collection of non-word characters that is recognized in the grammar of a language to hold some significance. All of our keywords right now have patterns that essentially match a word or multiple words. The block patterns don't actually have the literal string `{blocktype}Block` included in them; they are logic for grabbing all the text within the curly braces that follow the `graphql`, `sql`, `json`, or `csv` keyword.